fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,14 @@
+Your media, on your terms.
+
+The Jellyfin project is an open source, free software media server. No fees, no tracking, no hidden agenda. Get our free server to collect all your audio, video, photos, and more in one place.
+
+To use the app, you must have a Jellyfin server set up and running. Find out more at <a href='https://jellyfin.org/' target='_blank'>jellyfin.org</a>.
```

Comment:
If markdown is possible:
```suggestion
To use the app, you must have a Jellyfin server set up and running. Find out more at [jellyfin.org](https://jellyfin.org/).
```

Otherwise if we can use double-quotes in the HTML, that would be preferred:

```suggestion
To use the app, you must have a Jellyfin server set up and running. Find out more at <a href="https://jellyfin.org/" target="_blank">jellyfin.org</a>.
```

Reply:
While Markdown is possible with **my** repo, it would not be interpreted correectly by others (including F-Droid) – hence I intentionally used HTML in these places. As you target F-Droid in the end (and might wish to use it with Play Store as well), I'd suggest leaving it at that.